plugins/woocommerce/src/Internal/Admin/Translations.php

```diff
@@ -20,6 +20,13 @@ class Translations {
 	 */
 	protected static $instance = null;
 
+	/**
+	 * Plugin domain.
+	 *
+	 * @var string
+	 */
+	private static $plugin_domain = 'woocommerce';
```

Comment:
Just wondering outloud, is there any other potential value for `explode( '/', plugin_basename( __FILE__ ) )[0]` that would make sense? We are essentially hard coding this value instead of using the filepath to derive the value. Maybe in the case of a fork is all I can think of, but I'm curious if there is a reason why this wasn't hard coded in the first place.

Reply:
Good question! Yeah, I think we used it mainly because the text domain used to be woocommerce or woocommerce-admin, and filepath seemed to be a typical pattern to get the plugin slug/text domain. It wasn't until we used the live branch that many issues were discovered.

I can't think of any other value since text domain should be constant. It doesn't make sense to derive it from the filepath.